Applications/Calculator/CalculatorWindow.gml

```diff
@@ -0,0 +1,272 @@
+@GUI::Widget {
+    fixed_width: 254
+    fixed_height: 213
+    fill_with_background_color: true
+
+    layout: @GUI::VerticalBoxLayout {
+        margins: [10,0,10,0]
+    }
+
+    @GUI::TextBox {
+        name: "entry_textbox"
+    }
+
+    @GUI::Widget {
+        layout: @GUI::HorizontalBoxLayout {}
```

Comment:
In Libraries/LibGUI/FontPickerDialog.gml (lines 41 and 42), this is written with the following style; let's be consistent.  
```suggestion
        layout: @GUI::HorizontalBoxLayout {
        }
```